### ISSUE TITLE
Acute abdomen

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6163]
**Preferred term label:**
Acute abdomen

**Synonyms**
Acute abdominal pain 

**Definition (free text, please give PubMed ID)**
A sudden onset of abdominal pain with associated nausea or vomiting. The acute abdomen may be caused by an infection, inflammation, vascular occlusion, or obstruction.
PMID: 29083722 

**Parent term (use hpo.jax.org/app)**
Abdominal pain

**Comment by pnrobinson** [comment]
added
